package.json

```diff
@@ -186,6 +187,6 @@
   },
   "coordinates": [
     55.778264,
-    12.593202
```

Comment:
This change is probably just due to a merge conflict. Should just be reverted.

Reply:
Fixed